package.json

```diff
@@ -146,8 +147,8 @@
     "ts-node": "^9.0.0",
     "tsconfig-paths": "^3.9.0",
     "tsconfig-paths-webpack-plugin": "^3.3.0",
-    "webpack-bundle-analyzer": "^3.8.0",
-    "webpack-cli": "^3.3.12",
+    "webpack-bundle-analyzer": "^3.9.0",
+    "webpack-cli": "^4.1.0",
     "webpack-dev-server": "^3.11.0",
```

Comment:
Wont block the PR on it, but as `webpack serve` is being used, is dev server still needed? Perhaps worth removing in the next PR if so?